tests/network/sriov.go

```diff
@@ -520,6 +523,42 @@ var _ = Describe("[Serial]SRIOV", func() {
 			})
 		})
 
+		Context("Connected to multiple SRIOV networks", func() {
+			sriovNetworks := []string{sriovnet1, sriovnet2, sriovnet3, sriovnet4}
+			BeforeEach(func() {
+				for _, sriovNetwork := range sriovNetworks {
+					Expect(createSriovNetworkAttachmentDefinition(sriovNetwork, util.NamespaceTestDefault, sriovConfNAD)).To(Succeed(), shouldCreateNetwork)
+				}
+			})
+
+			It("should correctly plug all the interfaces based on the specified MAC and (guest) PCI addresses", func() {
+				macAddressTemplate := "de:ad:00:be:ef:%02d"
+				pciAddressTemplate := "0000:%02d:00.0"
+				vmi := getSriovVmi(sriovNetworks, defaultCloudInitNetworkData())
+				for i := range sriovNetworks {
+					vmi.Spec.Domain.Devices.Interfaces[i+1].PciAddress = fmt.Sprintf(pciAddressTemplate, i)
+					vmi.Spec.Domain.Devices.Interfaces[i+1].MacAddress = fmt.Sprintf(macAddressTemplate, i)
+				}
+
+				vmi = startVmi(vmi)
+				vmi = waitVmi(vmi)
+
+				Eventually(func() error {
+					for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
```

Comment:
I think we need to check if the interface is of SR-IOV type, looks like it failed due to not finding the MAC address probably for the primary NIC 
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/8218/pull-kubevirt-e2e-kind-1.22-sriov/1554042038645690368

Reply:
Added.